formal_singlepage.rst

```diff
@@ -0,0 +1,36 @@
+.. _formal_singlepage:
+
+.. container:: heading1
+
+    Normative CellML 2.0 Specification
+
+This document is the normative version of the CellML Specification, defining the CellML syntax and the rules by which it should be used.
+It is intended primarily for the developers of software tools which directly consume CellML syntax.
+
+Users of CellML models should read the informative version of the CellML Specification which is available at https://cellml.org/specifications/cellml_2.0.
```

Comment:
Why make it look like https://cellml.org/specifications/cellml_2.0 is only about the informative version of the CellML specification? Why not simply something like:

```
Additional information (e.g., a PDF version of this page, an informative version of the CellML Specification) can be found at https://cellml.org/specifications/cellml_2.0.
```

Reply:
That was the wording change we made in the meeting ... I just copied it to here too?

Reply:
Sure, it's just that now seeing it in context, I feel like it's not quite right. Others to comment though.